netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -67,13 +82,36 @@ public TkStart(final Base bse) {
 
     @Override
     public Response act(final Request req) throws IOException {
-        final Inbox inbox = new RqAlias(this.base, req).alias().inbox();
-        final long number = inbox.start();
+        final Href href = new RqHref.Base(req).href();
+        final Iterator<String> token = href.param("token").iterator();
+        String tokenkey = "";
+        final Inbox inbox;
+        long number = 0;
```

Comment:
@prahladyeri 
Exactly, you have to revert this here and create a new PR for the same
> As a side note, I've also clubbed the fix for #794 assigned to me in this PR (it is a harmless text change in README.md). Please let me know whether its fine, or I have to revert this and create a separate PR for that.